src/NserviceBus.Transport.Msmq/MsmqTransport.cs

```diff
@@ -37,7 +38,7 @@ public override TransportInfrastructure Initialize(SettingsHolder settings, stri
                 throw new Exception("Faults forwarding requires an error queue to be specified using 'EndpointConfiguration.SendFailedMessagesTo()'");
             }
 
-            settings.EnableFeature(typeof(InstanceMappingFileFeature));
+            settings.Set(typeof(InstanceMappingFileFeature).FullName, FeatureState.Enabled);
```

Comment:
Given that this is now in the MSMQ project we can just enable the feature by default  and remove this code?